### PR TITLE
[MB-9182] add new tf104 image, which updates tf and tfsec

### PR DESCRIPTION
**Pull request by sheenamt**
# Description

Rather than overwrite the tf103 image in order to update tfsec, we're using the sidecar approach. tf104 will be rolled out and in the case of breaking changes, tf103 can easily be subbed back in. 

## Changelog or Releases

- [terraform](https://github.com/hashicorp/terraform/releases)
- [tfsec](https://github.com/tfsec/tfsec/releases)

### CODE REVIEW

File: milmove-infra-tf104/tfsec-linux-amd64 [hunk 1]
```diff
@@ -0,0 +1 @@
+Not Found
```
Comment (carterjones) [inline milmove-infra-tf104/tfsec-linux-amd64:1]: This seems like it may be leftover from something. Was this file intended to be here? Or perhaps was it supposed to have other content?

**Review by CoryAtTruss — APPROVED** [review]
